File: tests/dynamic_fixtures/java/cmdi_unsupported.java
// Command injection — unsupported fixture.
// Entry is an instance method; test sets confidence = Low.
// Expected verdict: Unsupported

import java.io.*;

public class Entry {
    public void execute(String cmd) throws Exception {
        Runtime.getRuntime().exec(new String[]{"/bin/sh", "-c", cmd});
    }
}

File: tests/dynamic_fixtures/go/cmdi_unsupported.go
// Command injection — unsupported fixture.
// Entry is a method on a struct.
// Test sets confidence = Low to get Unsupported(ConfidenceTooLow).
// Entry: Runner.Execute  Cap: CODE_EXEC
// Expected verdict: Unsupported

package entry

import "os/exec"

type Runner struct{}

func (r *Runner) Execute(cmd string) {
	exec.Command("sh", "-c", cmd).Run()
}
